anaconda3/amazonlinux/Dockerfile

```diff
@@ -8,7 +8,7 @@ ENV PATH /opt/conda/bin:$PATH
 RUN yum install -y wget bzip2 ca-certificates libglib2.0-0 libxext6 libsm6 libxrender1 git mercurial subversion && \
```

Comment:
The amazon image could be also updated to `arm64v8/amazonlinux:2.0.20211005.0`, see beginning of the file. The anaconda-pkg-build image, just uses `amazonlinux:2`.